src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -43,7 +45,8 @@ public Path getCloudSdkPath() {
 
     // search program files
     if (System.getProperty("os.name").contains("Windows")) {
-      possiblePaths.add(getProgramFilesLocation());
+        possiblePaths.add(getLocalAppDataLocation());
```

Comment:
Weren't we just using the 'where/which' commands for this?  Or is that some place else and this is a fallback?